midas/session_creator.py

```diff
@@ -0,0 +1,10 @@
+import os
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+DB_PATH = os.path.join(os.path.dirname(__file__), 'midas.db')
+
+Base = declarative_base()
+engine = create_engine(f'sqlite:///{DB_PATH}')
+session = sessionmaker(bind=engine)()
```

Comment:
It'll be very weird to use the `session` like that.
Usually, we tend to close it after we're done, and in this form of implementation it'll be impossible to open a new session after we've closed the first one.
Try and work with a context manager here.